scripts/run-todomvc-in-ios-sim.sh

```diff
@@ -2,7 +2,8 @@
 set -euo pipefail
 
 function cleanup {
-  if [ -n "$uuid" ]; then
+  # alternative to [ -v ] for bash prior to 4.2
+  if [ -n "${uuid-}" ]; then
```

Comment:
With `set -euo` I get an expansion error here when the script fails before launching the simulator